spec/core/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -610,8 +624,31 @@ function recvPacket(
     abortTransactionUnless(connection !== null)
     abortTransactionUnless(connection.state === OPEN)
 
-    abortTransactionUnless(packet.timeoutHeight === 0 || getConsensusHeight() < packet.timeoutHeight)
-    abortTransactionUnless(packet.timeoutTimestamp === 0 || currentTimestamp() < packet.timeoutTimestamp)
+    switch channel.order {
+      case ORDERED:
```

Comment:
Is it possible to do something like? I get that this is a typescript fall through, but coming from a go lens it performs completely different behaviour 
```go
case ORDERED || UNORDERED:
```
or
```go
case ORDERED, UNORDERED:
```

Reply:
I'd use the first variant, i.e., `case ORDERED || UNORDERED`. Seems less ambiguous.